README.md

```diff
@@ -51,6 +51,7 @@ Maven Coordinates:
               <skip>false</skip><!-- this is the default -->
               <chatty>false</chatty><!-- this is the default -->
               <quiet>false</quiet><!-- this is the default -->
+              <redirect>true</redirect><!-- this is the default -->
```

Comment:
I don't think the default should be true. If this maven plugin is already used by anyone who actually asserts a redirect value this might break their tests.

Reply:
I think that following redirects is the current behavior, right?